client/components/common/MyBuilds.tsx

```diff
@@ -2,498 +2,19 @@
 
```

Comment:
I feel like you don't need this component anymore - it's totally identical to BuildList right now.